terraform/templates/local/ssm-install-aoc.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
```

Comment:
Actually we can convert those shell script into go code to make life easier. I haven't created the PR for https://github.com/aws-observability/aws-otel-test-framework/issues/255 it has some ssm wrapper for patching related ssm status query.

Reply:
Agree. Golang will make error handling easier. No matter what language, the logic of async execution is required. The testcase should fail when any error occurs. We can use bash script for now and convert its logic to golang later.